lib/bridge-client/index.js

```diff
@@ -790,9 +789,12 @@ BridgeClient.prototype._shardTransferComplete = function(state, frame, done) {
       state.callback(err, file);
       return done();
     });
-  }
+  };
 
-  return _shardTransferComplete();
+  state.cleanup(function(err) {
+    if(err) return done(err || null);
```

Comment:
If we encounter an error during cleanup, we should probably still register the file on the bridge by calling `_shardTransferComplete();`. The inability to cleanup temporary files doesn't affect whether or not the transfer was successful.